docs/guide/input-file-upload.md

```diff
@@ -55,6 +55,12 @@ the [Core Validators](tutorial-core-validators.md#file) section for more details
   implemented via [[yii\validators\ImageValidator]] which verifies if an attribute has received a valid image 
   that can be then either saved or processed using the [Imagine Extension](https://github.com/yiisoft/yii2-imagine).
 
+If you need to validate your model after use of `saveAs` method, especially with the `image` validator, you need to add `false` paramether to `saveAs`, because this method deletes an uploaded file during saving by default and validator will give out an error.
```

Comment:
> The bad and too vague note.
May be. But if I found out this option earlier in this guide, it would have saved me from hours of debug and invent of a bicycles.